components/prism-log.js

```diff
@@ -93,7 +93,7 @@ Prism.languages.log = {
 		alias: 'number'
 	},
 	'time': {
-		pattern: /\b\d{1,2}:\d{1,2}:\d{1,2}(?:[.,:]\d+)?(?:\s?[+-]\d{2,4}|Z)?\b/,
+		pattern: /\b[T\s]?\d{1,2}:\d{1,2}:\d{1,2}(?:[.,:]\d+)?(?:\s?[+-]\d{2}:?\d{2}|Z)?\b/,
```

Comment:
Since `T` is part of the date, it shouldn't be part the time.

```suggestion
		pattern: /\b\d{1,2}:\d{1,2}:\d{1,2}(?:[.,:]\d+)?(?:\s?[+-]\d{2}:?\d{2}|Z)?\b/,
```